FreeBSD: Missing include <sstream> in metadata.cc

### Describe the bug (and expected behaviour)

Hello,

Updating goldendict-ng to latest release and build fails.
Fixed by adding `#include <sstream>` to `src/metadata.cc`
 
```
FAILED: [code=1] CMakeFiles/goldendict-ng.dir/src/metadata.cc.o
/usr/bin/c++ -DLIBICONV_PLUG -DMAKE_CHINESE_CONVERSION_SUPPORT -DMAKE_QTMULTIMEDIA_PLAYER -DMAKE_ZIM_SUPPORT -DPROGRAM_DATA_DIR=\"/usr/local/share/goldendict-ng\" -DPROGRAM_VERSION=\"26.1.1\" -DQT_CONCURRENT_LIB
-DQT_CORE5COMPAT_LIB -DQT_CORE_LIB -DQT_DISABLE_DEPRECATED_UP_TO=0x060700 -DQT_ENABLE_STRICT_MODE_UP_TO=0x060800 -DQT_GUI_LIB -DQT_MULTIMEDIA_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_NO_DEPRECATED_WARNINGS -DQT_OP
ENGL_LIB -DQT_POSITIONING_LIB -DQT_PRINTSUPPORT_LIB -DQT_QMLINTEGRATION_LIB -DQT_QML_LIB -DQT_QUICK_LIB -DQT_SVG_LIB -DQT_WEBCHANNEL_LIB -DQT_WEBENGINECORE_LIB -DQT_WEBENGINEWIDGETS_LIB -DQT_WIDGETS_LIB -DQT_XML_
LIB -DWITH_X11 -I/wrkdirs/usr/ports/textproc/goldendict-ng/work/.build/goldendict-ng_autogen/include -I/wrkdirs/usr/ports/textproc/goldendict-ng/work/goldendict-ng-26.1.1-Release.5491ffca/thirdparty/qtsingleappli
cation/src -I/wrkdirs/usr/ports/textproc/goldendict-ng/work/goldendict-ng-26.1.1-Release.5491ffca/src -I/wrkdirs/usr/ports/textproc/goldendict-ng/work/goldendict-ng-26.1.1-Release.5491ffca/src/common -I/wrkdirs/u
sr/ports/textproc/goldendict-ng/work/goldendict-ng-26.1.1-Release.5491ffca/src/dict -I/wrkdirs/usr/ports/textproc/goldendict-ng/work/goldendict-ng-26.1.1-Release.5491ffca/src/dict/utils -I/wrkdirs/usr/ports/textp
roc/goldendict-ng/work/goldendict-ng-26.1.1-Release.5491ffca/src/ui -isystem /usr/local/include/qt6/QtCore -isystem /usr/local/include/qt6 -isystem /usr/local/lib/qt6/mkspecs/freebsd-clang -isystem /usr/local/inc
lude/qt6/QtXml -isystem /usr/local/include/qt6/QtConcurrent -isystem /usr/local/include/qt6/QtCore5Compat -isystem /usr/local/include/qt6/QtMultimedia -isystem /usr/local/include/qt6/QtGui -isystem /usr/local/inc
lude -isystem /usr/local/include/qt6/QtNetwork -isystem /usr/local/include/qt6/QtPrintSupport -isystem /usr/local/include/qt6/QtWidgets -isystem /usr/local/include/qt6/QtWebEngineWidgets -isystem /usr/local/inclu
de/qt6/QtWebEngineCore -isystem /usr/local/include/qt6/QtQuick -isystem /usr/local/include/qt6/QtQml -isystem /usr/local/include/qt6/QtQmlIntegration -isystem /usr/local/include/qt6/QtOpenGL -isystem /usr/local/i
nclude/qt6/QtWebChannel -isystem /usr/local/include/qt6/QtPositioning -isystem /usr/local/include/qt6/QtSvg -isystem /usr/local/include/hunspell -isystem /usr/local/include/lzo -isystem /usr/local/include/opencc
-O2 -pipe -fstack-protector-strong -fno-strict-aliasing -O2 -pipe -fstack-protector-strong -fno-strict-aliasing    -DNDEBUG -std=gnu++17 -fPIC -pthread -D_THREAD_SAFE -DTOML_HEADER_ONLY=0 -DTOML_SHARED_LIB=1 -MD
-MT CMakeFiles/goldendict-ng.dir/src/metadata.cc.o -MF CMakeFiles/goldendict-ng.dir/src/metadata.cc.o.d -o CMakeFiles/goldendict-ng.dir/src/metadata.cc.o -c /wrkdirs/usr/ports/textproc/goldendict-ng/work/goldendi
ct-ng-26.1.1-Release.5491ffca/src/metadata.cc
/wrkdirs/usr/ports/textproc/goldendict-ng/work/goldendict-ng-26.1.1-Release.5491ffca/src/metadata.cc:77:23: error: implicit instantiation of undefined template 'std::basic_stringstream<char>'
   77 |     std::stringstream ss;
      |                       ^
/usr/include/c++/v1/__fwd/sstream.h:30:28: note: template is declared here
   30 | class _LIBCPP_TEMPLATE_VIS basic_stringstream;
      |                            ^
1 error generated.
```

Thanks

### Extra Info (logs, crash dumps, malfunctioning dictionaries...)

_No response_

### OS and software versions

FreeBSD
goldendict-ng-v26.1.1-Release.5491ffca

--- a/src/metadata.cc
+++ b/src/metadata.cc
@@ -3,6 +3,9 @@
 #include <QDebug>
 #include <QSaveFile>
 #include <QFile>
+#ifdef Q_OS_FREEBSD
+#include <sstream>
+#endif
 
 std::optional< Metadata::result > Metadata::load( std::string_view filepath )
 {

--- a/src/metadata.cc
+++ b/src/metadata.cc
@@ -4,7 +4,7 @@
 #include <QSaveFile>
 #include <QFile>
 #ifdef Q_OS_FREEBSD
-#include <sstream>
+  #include <sstream>
 #endif
 
 std::optional< Metadata::result > Metadata::load( std::string_view filepath )

